app/views/candidate_mailer/application_under_consideration.text.erb

```diff
@@ -0,0 +1,15 @@
+Dear <%= @candidate_name %>,
+
+# Your application is being considered
+
+Your application is now with your teacher training provider(s).
+
+We’ve asked providers to make a final decision within <%= @reject_by_default_days %> working days.
```

Comment:
I think we should be giving the actual date now we know it rather than "40 working days" which isn't very helpful. cc @paulrobertlloyd

Reply:
Yup, `40 workings day (1 February 2020)` or something similar.

Reply:
`They need to make a final decision before 1 February 2020 (40 working days)`